lib/commands/actions.js

```diff
@@ -2,17 +2,24 @@ import androidHelpers from '../android-helpers';
 import { fs, util, tempDir} from '@appium/support';
 import path from 'path';
 import B from 'bluebird';
-import sharp from 'sharp';
 import { exec } from 'teen_process';
 import { requireArgs } from '../utils';
 
 const swipeStepsPerSec = 28;
 const dragStepsPerSec = 40;
+let _sharp;
 
 const commands = {};
 const helpers = {};
 const extensions = {};
 
+function requireSharp() {
+  if (!_sharp) {
+    _sharp = require('sharp');
```

Comment:
do we want to wrap this in error handling with a nice error message so the user knows what to do if this fails?

Reply:
or maybe this `requireSharp` function can even just be part of appium support? isn't sharp already an optional dep there?

Reply:
it is already there